lib/xapi/implementation.rb

```diff
@@ -91,6 +91,8 @@ def assemble_readme
 #{readme_body}
 
 #{problem.source_markdown}
+
+#{incomplete_solutions_body}
```

Comment:
@Insti fyi - there's a discussion on the issue: https://github.com/exercism/discussions/issues/54
I think it would be great to monitor/comment/tweak how things are going there, and I'd really value your input.